File: 08-19/Main.java
import adts.list.*;

public class Main {
  public static void main(String[] args) {
    List<Integer> l = new LinkedList<Integer>();

    l.add(0);
    l.add(1);
    l.add(2);
    l.add(3);

    for (int i = 0; i < 5; i++) {
      System.out.println(l.contains(i));
    }
  }

}

File: 08-19/main.cpp
#include "./adts/list/linked_list.cpp"
#include <iostream>

int main()
{
  List<int> *l = new LinkedList<int>();

  l->add(0);
  l->add(1);
  l->add(2);
  l->add(3);

  for (int i = 0; i < 5; i++)
  {
    std::cout << l->contains(i) << std::endl;
  }

  return 0;
}
